bin/sync-config

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+/**
+ * This node js executable reads in the top-level package.json, babel.config.js
+ * and the special .eslintrc.track.json and copies them to each exercise.
```

Comment:
Can the Makefile do this? Since it already copies `package.json`, `ASSIGNMENT` and `TEST_FILE`.

Reply:
Check the other PR to see where this is going.

Reply:
(so yes, but I rather have ALL functionality in Make or ALL in node/sh, not all over the place. I don't know how to parse json in Make, remove a key and do a comparison)

Reply:
https://stedolan.github.io/jq/

Reply:
The point is that we're forcing contributors to learn and install Make when everything else is in JavaScript. We're using npm as package manager and to get the dependencies in but then use another system to run scripts?